packages/neuron-wallet/src/locales/en.ts

```diff
@@ -206,6 +206,14 @@ export default {
         'fail-message':
           'Unable to launch mail client, please copy the mail address, append debug information exported by "Menu" -> "Help" -> "Export Debug Information" and send us.',
         'copy-mail-addr': 'Copy mail address'
+      },
+      'migrate-failed': {
+        title: 'Migrate failed',
+        message:
+          'Migrate failed, press ok to delete old data and synchronize from scratch, or back-up old data and then press ok.',
```

Comment:
As I asked before, the `back-up old data` is useless if the migration fails because data will be broken at that time, we should request the user to confirm the backup before migration.